src/qml/editorwidgets/ExternalResource.qml

```diff
@@ -77,6 +77,26 @@ EditorWidgetBase {
     if (currentValue != undefined && currentValue !== '') {
       const isHttp = value.startsWith('http://') || value.startsWith('https://');
       var fullValue = isHttp ? value : prefixToRelativePath + value;
+      const fullValueExists = FileUtils.fileExists(fullValue);
+      if (!fullValueExists && externalStorage.type != "") {
+        prepareValue("");
+        if (config["StorageAuthConfigId"] !== "" && !iface.isAuthenticationConfigurationAvailable(config["StorageAuthConfigId"])) {
+          mainWindow.displayToast(qsTr("The external storage's authentication configuration ID is missing, please insure it is imported into QField"), "error", qsTr("Learn more"), function () {
+              Qt.openUrlExternally('https://docs.qfield.org/how-to/authentication/');
```

Comment:
Relevant PR that improves this page: https://github.com/opengisch/QField-docs/pull/535